matlab/src/cpp/arrow/matlab/array/proxy/array.cc

```diff
@@ -23,7 +23,7 @@ namespace arrow::matlab::array::proxy {
 
         // Register Proxy methods.
         REGISTER_METHOD(Array, ToString);
-        REGISTER_METHOD(Array, ToMatlab);
+        REGISTER_METHOD(Array, toMATLAB);
```

Comment:
I'm not familiar with MATLAB API but is it natural that `ToString` uses `T` and `toMATLAB` uses `t`?

Reply:
Hi @kou, That's a mistake on our part. `ToString` should be `toString`. Typically, we use lower camel case for MATLAB method names. Thanks for catching this!

Reply:
1. Changed `ToString` to `toString`.
2. Also renamed  the c++ method `Length()` `length()`. However, since this `Length` is a property on the MATLAB `arrow.array.Array` class, I left the property name alone. In MATLAB, property names are typically upper camel case.